src/context/talentLayer.tsx

```diff
@@ -113,8 +122,9 @@ const TalentLayerProvider = ({ children }: { children: ReactNode }) => {
       refreshData: fetchData,
       loading,
       completionScores,
+      talentLayerClient
```

Comment:
so that's good to have it here now, next steps use it everywhere rather than using the hook

Reply:
This was actually a conscious choice to make migration to react-sdk eaiser

Details in this comment: https://github.com/TalentLayer-Labs/starter-kit/pull/21/files#r1348862824